src/main/pages/che-7/end-user-guide/proc_using-different-types-of-storage.adoc

```diff
@@ -0,0 +1,62 @@
+---
+title: Using different type of storage 
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/using-diffrent-type-of-storage/
+folder: che-7/end-user-guide
+summary:
+---
+
+[id="using-diffrent-type-of-storage_{context}"]
+= Using different type of storage 
+
+Since 7.17.0 Eclipse Che supports three types of storage with different capabilities: persistent storage, ephemeral and asynchronous.
+
+.Persistent storage
+
+This kind of storage gives you an ability to store your changes directly in the mounted Persistent Volume. In this case your changes will be totally safe and Kubernetes infrastructure takes care of it (storage backend in particular), but the price for that is slow I/O, especially with a huge amount of small files. For example, some NodeJS projects may have many dependencies and node_modules is filled with thousands of small files.
+
+For now this is the default mode for any new workspace, no additional attributes are required. If you want to explicitly make this setting visible in your workspace configuration you can add:
+[source,yaml]
+----
+attributes:
+ persistVolumes: ‘true’
+----
+
+.Ephemeral Storage
+
+With this kind of storage files are mounted to emptyDir. As the name says, it is initially empty. When a Pod is removed from a node for any reason, the data in the emptyDir is deleted forever.  This means that you lose all your changes if you stop your workspace (or workspace pod restarts for any reason).
```

Comment:
lets's add a link to the official kubernetes docs - https://kubernetes.io/docs/concepts/storage/volumes/#emptydir

Reply:
I would also put emptyDir in the `` to outline the term e.g  - `emptyDir` volume

Reply:
ok